lib/rules/structure/h2.js

```diff
@@ -3,18 +3,8 @@ exports.name = "structure.h2";
 exports.check = function (sr, done) {
     var h2s = [];
     sr.$("h2").each(function () {
-        var $h2 = sr.$(this)
-        ,   inHead = false
-        ;
-        // whacko doesn't really do .parents(selector)
-        $h2.parents().each(function () {
-            var $el = sr.$(this);
-            if (($el[0].name || $el[0].tagName).toLowerCase() === "div" && $el.hasClass("head")) {
-                inHead = true;
-                return false;
-            }
-        });
-        if (!inHead) h2s.push(sr.norm($h2.text()));
+        var $h2 = sr.$(this);
+        if ($h2.parents("div.head").length === 0) h2s.push(sr.norm($h2.text()));
```

Comment:
There might be a better way to filter out the `h2` elements that are in the `div.head`, in which case I am all ears.